roles/kubevirt-apb/tasks/deprovision.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Deleting DaemonSet libvirt
+  k8s_v1beta2_daemon_set:
+    name: libvirt
+    namespace: "{{ namespace}}"
+    state: absent
```

Comment:
The difference between deprovision and provision is the state variable.  We could combine this into a single template and make a variable for ```"{{ state }}"```.

Reply:
while i personally like this kind of abstraction, i believe end users will have a harder time checking the code with this kind of (elegant) trick

Reply:
It's not too complicated. We just add ```state = present``` to ```provision.yml```.  

```yaml
...
    install_python_requirements: no
  - role: ansibleplaybookbundle.asb-modules
  - role: kubevirt-apb
  vars:
    state: present
...
```

But that being said, I'm fine with keeping the separation between deprovision and provision for now because I think this app may require additional action that are unique to each playbook. Let's leave it and we can combine later if the delta doesn't change.